include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -189,68 +203,124 @@ struct BFloat16_t {
 
 static_assert(sizeof(BFloat16_t) == sizeof(uint16_t), "Sizes must match");
 
-/** \brief Used internally by the C++ API. C++ wrapper types inherit from this
+namespace detail {
```

Comment:
I see detail:: everywhere that it didn't use to be, as this increases the amount to read, how is this better?

Reply:
Well, `detail` is one of the standard techniques to designate and hide implementation details.
For example, in IDE a use would type `Ort::`, types within detail would not be shown as a suggestion. Certainly Base<T> should have never been a type in `Ort`.

Reply:
Now I know